src/pint/simulation.py

```diff
@@ -85,13 +82,11 @@ def zero_residuals(ts, model, maxiter=10, tolerance=None):
         )
 
 
-def update_fake_toa_clock(ts, model, include_bipm=False, include_gps=True):
-    """Update the clock settings (corrections, etc) for fake TOAs
+def get_fake_toa_clock_versions(model, include_bipm=False, include_gps=True):
```

Comment:
Can this share code with the version in `pint.toa`?

Reply:
That might be better - I was fixing what Jing had.  I'll see how straightforward it is.